tools/toolchain/Dockerfile

```diff
@@ -4,6 +4,10 @@ ARG CLANG_BUILD="SKIP"
 ARG CLANG_ARCHIVE
 
 WORKDIR /mnt
+ADD ./install-dependencies.sh ./
+ADD ./seastar/install-dependencies.sh ./seastar/
+ADD ./tools/java/install-dependencies.sh ./tools/java/
+ADD ./tools/toolchain/system-auth ./
```

Comment:
probably this deserves its dedicated PR? also i cannot find the explanations regarding this change in the commit message. also, if we want to run the local script on host, probably we could use `RUN --mount=type=bind`, like:
```Dockerfile
RUN --mount=type=bind,source=.,target=./scylla dnf -y update \
    && dnf -y install 'dnf-command(copr)' \
    && dnf -y install ccache \
    && dnf -y install devscripts debhelper fakeroot file rpm-build \
    && scylla/install-dependencies.sh && dnf clean all \
    && echo 'ALL ALL=(ALL:ALL) NOPASSWD: ALL' >> /etc/sudoers \
    && cp scylla/tools/toolchain/system-auth /etc/pam.d \
    && echo 'Defaults !requiretty' >> /etc/sudoers
RUN mkdir -p /root/.m2/repository
ENV JAVA8_HOME=/usr/lib/jvm/java-1.8.0-openjdk
RUN scylla/tools/toolchain/optimized_clang.sh
```

unless we need to access these scripts in the container after building the image.

Reply:
Also this one too, fixed.